### DISCUSSION TITLE
CS Student from Malaysia: Excited to Learn Git and GitHub!

### DISCUSSION THREAD

**Discussion by sharin-io** [community/community#135742] [New to GitHub] [answered]
Hi all! I'm Sharin Bi👩🏻‍💻, and I'm excited to join the GitHub Community on Discussions!
A little about me:

- I'm currently studying Computer Science, majoring in Software Development at Swinburne Technology University Course at INTI International College Subang.

- My main interest right now is self-studying Git and GitHub, as well as other beginner-friendly programming languages.

- I'm looking forward to exploring all the different areas of this community and learning more about using GitHub effectively.

I'm curious to know if anyone else is experiencing a 404 error like I am? If so, I'd really appreciate any advice on how to solve this issue.
Looking forward to connecting with fellow developers and learning together!
<img width="1440" alt="Screenshot 2024-08-14 at 12 15 00 PM" src="https://github.com/user-attachments/assets/9986d85f-77cb-4da4-b2e2-dfb16e492544">

**Comment by abdimk** [comment]
Hi **Sharin Bi** 👩🏻‍💻! Welcome to the GitHub Community and Discussions! 🎉 It's great to have you here. Whether you're looking to collaborate on projects, share knowledge, or learn from others, the community is a fantastic place to connect with like-minded developers. Feel free to ask questions, contribute to ongoing discussions, or start new ones. We're excited to see what you'll bring to the community! 🚀

**Version control** is a crucial tool in modern software development, providing structure and safety when managing changes to code. Git is a powerful and flexible version control system, and GitHub enhances Git with a collaborative platform that connects developers around the world. Whether you're working on a personal project or contributing to an open-source initiative, understanding Git and GitHub is essential for effective collaboration and project management.

What is Git? Our beginner’s guide to version control
https://github.blog/developer-skills/programming-languages-and-frameworks/what-is-git-our-beginners-guide-to-version-control/

Git and GitHub for Beginners - Crash Course
https://www.youtube.com/watch?v=RGOj5yH7evk

**Comment by ghostinhershell** [comment]
👋 Welcome to the GitHub Community on Discussions, @sharin-io! Happy to have you here 🙂

To get started, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776).

**Comment by ibrartalab** [comment, marked as answer]
@sharin-io  Hi my name is ibrar yesterday i was facing the same issue here is the correct route for what you are looking:
https://education.github.com/learner/learn

**Reply by sharin-io** [reply]
Thank you for referring me to the link. I just solved it !

**Comment by sojanonelson** [comment]
Hi @sharin-io , welcome to the GitHub Community!

It's great that you're diving into Git and GitHub! Regarding the 404 error, it usually happens when the page or resource you're trying to access doesn't exist, or there's a permission issue (e.g., accessing a private repository or an incorrect link).

Try these steps:
1. **Double-check the URL** for typos or if the resource exists.
2. **Check your permissions** if it's a private repo (ensure you're added as a collaborator).
3. **Clear your cache** or try accessing from a different browser.

Feel free to share more details if the issue persists. Looking forward to seeing your contributions!
